 Network vhogemann_default  Creating
 Network vhogemann_default  Created
 Container vhogemann-postgres-1  Creating
 Container vhogemann-postgres-1  Created
 Container vhogemann-api1-1  Creating
 Container vhogemann-api2-1  Creating
 Container vhogemann-api2-1  Created
 Container vhogemann-api1-1  Created
 Container vhogemann-nginx-1  Creating
 Container vhogemann-nginx-1  Created
Attaching to api1-1, api2-1, nginx-1, postgres-1
postgres-1  | The files belonging to this database system will be owned by user "postgres".
postgres-1  | This user must also own the server process.
postgres-1  | 
postgres-1  | The database cluster will be initialized with locale "en_US.utf8".
postgres-1  | The default database encoding has accordingly been set to "UTF8".
postgres-1  | The default text search configuration will be set to "english".
postgres-1  | 
postgres-1  | Data page checksums are disabled.
postgres-1  | 
postgres-1  | fixing permissions on existing directory /var/lib/postgresql/data ... ok
postgres-1  | creating subdirectories ... ok
postgres-1  | selecting dynamic shared memory implementation ... posix
postgres-1  | selecting default max_connections ... 100
postgres-1  | selecting default shared_buffers ... 128MB
postgres-1  | selecting default time zone ... Etc/UTC
postgres-1  | creating configuration files ... ok
postgres-1  | running bootstrap script ... ok
postgres-1  | performing post-bootstrap initialization ... ok
postgres-1  | syncing data to disk ... ok
postgres-1  | 
postgres-1  | 
postgres-1  | Success. You can now start the database server using:
postgres-1  | 
postgres-1  |     pg_ctl -D /var/lib/postgresql/data -l logfile start
postgres-1  | 
postgres-1  | initdb: warning: enabling "trust" authentication for local connections
postgres-1  | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
postgres-1  | waiting for server to start....2024-02-19 17:10:44.255 UTC [48] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
postgres-1  | 2024-02-19 17:10:44.258 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres-1  | 2024-02-19 17:10:44.268 UTC [51] LOG:  database system was shut down at 2024-02-19 17:10:42 UTC
postgres-1  | 2024-02-19 17:10:44.274 UTC [48] LOG:  database system is ready to accept connections
postgres-1  |  done
postgres-1  | server started
postgres-1  | 
postgres-1  | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/bootstrap.sql
postgres-1  | CREATE TABLE
postgres-1  | CREATE INDEX
postgres-1  | CREATE TYPE
postgres-1  | CREATE TABLE
postgres-1  | CREATE INDEX
postgres-1  | CREATE INDEX
nginx-1     | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1     | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
postgres-1  | CREATE PROCEDURE
postgres-1  | CREATE PROCEDURE
postgres-1  | INSERT 0 6
postgres-1  | 
postgres-1  | 
postgres-1  | waiting for server to shut down....2024-02-19 17:10:44.439 UTC [48] LOG:  received fast shutdown request
postgres-1  | 2024-02-19 17:10:44.443 UTC [48] LOG:  aborting any active transactions
postgres-1  | 2024-02-19 17:10:44.445 UTC [48] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
postgres-1  | 2024-02-19 17:10:44.445 UTC [49] LOG:  shutting down
postgres-1  | 2024-02-19 17:10:44.447 UTC [49] LOG:  checkpoint starting: shutdown immediate
nginx-1     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1     | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx-1     | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1     | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1     | /docker-entrypoint.sh: Configuration complete; ready for start up
postgres-1  | 2024-02-19 17:10:44.647 UTC [49] LOG:  checkpoint complete: wrote 91 buffers (0.6%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.012 s, sync=0.179 s, total=0.202 s; sync files=73, longest=0.103 s, average=0.003 s; distance=268 kB, estimate=268 kB; lsn=0/152DAC0, redo lsn=0/152DAC0
postgres-1  | 2024-02-19 17:10:44.650 UTC [48] LOG:  database system is shut down
postgres-1  |  done
postgres-1  | server stopped
postgres-1  | 
postgres-1  | PostgreSQL init process complete; ready for start up.
postgres-1  | 
postgres-1  | 2024-02-19 17:10:44.765 UTC [1] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
postgres-1  | 2024-02-19 17:10:44.765 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
postgres-1  | 2024-02-19 17:10:44.765 UTC [1] LOG:  listening on IPv6 address "::", port 5432
postgres-1  | 2024-02-19 17:10:44.772 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres-1  | 2024-02-19 17:10:44.780 UTC [64] LOG:  database system was shut down at 2024-02-19 17:10:44 UTC
postgres-1  | 2024-02-19 17:10:44.787 UTC [1] LOG:  database system is ready to accept connections
postgres-1  | 2024-02-19 17:10:47.126 UTC [76] FATAL:  role "root" does not exist
postgres-1  | 2024-02-19 17:10:52.230 UTC [86] FATAL:  role "root" does not exist
postgres-1  | 2024-02-19 17:10:57.306 UTC [98] FATAL:  role "root" does not exist
postgres-1  | 2024-02-19 17:11:02.383 UTC [108] FATAL:  role "root" does not exist
postgres-1  | 2024-02-19 17:11:07.466 UTC [116] FATAL:  role "root" does not exist
postgres-1  | 2024-02-19 17:11:12.103 UTC [100] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:12.103 UTC [100] DETAIL:  Failing row contains (2, 2, -80136, 80000).
postgres-1  | 2024-02-19 17:11:12.103 UTC [100] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:12.103 UTC [100] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:12.544 UTC [124] FATAL:  role "root" does not exist
postgres-1  | 2024-02-19 17:11:13.203 UTC [99] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:13.203 UTC [99] DETAIL:  Failing row contains (1, 1, -101119, 100000).
postgres-1  | 2024-02-19 17:11:13.203 UTC [99] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:13.203 UTC [99] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:13.417 UTC [87] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:13.417 UTC [87] DETAIL:  Failing row contains (1, 1, -103172, 100000).
postgres-1  | 2024-02-19 17:11:13.417 UTC [87] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:13.417 UTC [87] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:15.348 UTC [68] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:15.348 UTC [68] DETAIL:  Failing row contains (2, 2, -82600, 80000).
postgres-1  | 2024-02-19 17:11:15.348 UTC [68] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:15.348 UTC [68] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:15.380 UTC [77] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:15.380 UTC [77] DETAIL:  Failing row contains (2, 2, -83540, 80000).
postgres-1  | 2024-02-19 17:11:15.380 UTC [77] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:15.380 UTC [77] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:15.670 UTC [100] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:15.670 UTC [100] DETAIL:  Failing row contains (2, 2, -86486, 80000).
postgres-1  | 2024-02-19 17:11:15.670 UTC [100] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:15.670 UTC [100] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:15.915 UTC [77] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:15.915 UTC [77] DETAIL:  Failing row contains (2, 2, -82827, 80000).
postgres-1  | 2024-02-19 17:11:15.915 UTC [77] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:15.915 UTC [77] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:16.036 UTC [89] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:16.036 UTC [89] DETAIL:  Failing row contains (2, 2, -86080, 80000).
postgres-1  | 2024-02-19 17:11:16.036 UTC [89] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:16.036 UTC [89] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:16.902 UTC [100] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:16.902 UTC [100] DETAIL:  Failing row contains (1, 1, -106807, 100000).
postgres-1  | 2024-02-19 17:11:16.902 UTC [100] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:16.902 UTC [100] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:16.960 UTC [87] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:16.960 UTC [87] DETAIL:  Failing row contains (2, 2, -84217, 80000).
postgres-1  | 2024-02-19 17:11:16.960 UTC [87] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:16.960 UTC [87] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:17.131 UTC [77] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:17.131 UTC [77] DETAIL:  Failing row contains (2, 2, -86259, 80000).
postgres-1  | 2024-02-19 17:11:17.131 UTC [77] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:17.131 UTC [77] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:17.417 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:17.417 UTC [88] DETAIL:  Failing row contains (2, 2, -87696, 80000).
postgres-1  | 2024-02-19 17:11:17.417 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:17.417 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:17.502 UTC [100] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:17.502 UTC [100] DETAIL:  Failing row contains (2, 2, -88113, 80000).
postgres-1  | 2024-02-19 17:11:17.502 UTC [100] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:17.502 UTC [100] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:17.515 UTC [68] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:17.515 UTC [68] DETAIL:  Failing row contains (2, 2, -80178, 80000).
postgres-1  | 2024-02-19 17:11:17.515 UTC [68] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:17.515 UTC [68] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:17.621 UTC [134] FATAL:  role "root" does not exist
postgres-1  | 2024-02-19 17:11:17.674 UTC [99] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:17.674 UTC [99] DETAIL:  Failing row contains (2, 2, -83756, 80000).
postgres-1  | 2024-02-19 17:11:17.674 UTC [99] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:17.674 UTC [99] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:17.757 UTC [99] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:17.757 UTC [99] DETAIL:  Failing row contains (1, 1, -105386, 100000).
postgres-1  | 2024-02-19 17:11:17.757 UTC [99] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:17.757 UTC [99] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:17.811 UTC [100] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:17.811 UTC [100] DETAIL:  Failing row contains (1, 1, -104853, 100000).
postgres-1  | 2024-02-19 17:11:17.811 UTC [100] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:17.811 UTC [100] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:17.892 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:17.892 UTC [88] DETAIL:  Failing row contains (1, 1, -100730, 100000).
postgres-1  | 2024-02-19 17:11:17.892 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:17.892 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:17.919 UTC [90] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:17.919 UTC [90] DETAIL:  Failing row contains (1, 1, -107631, 100000).
postgres-1  | 2024-02-19 17:11:17.919 UTC [90] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:17.919 UTC [90] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:18.000 UTC [77] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:18.000 UTC [77] DETAIL:  Failing row contains (1, 1, -106465, 100000).
postgres-1  | 2024-02-19 17:11:18.000 UTC [77] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:18.000 UTC [77] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:18.135 UTC [99] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:18.135 UTC [99] DETAIL:  Failing row contains (1, 1, -103654, 100000).
postgres-1  | 2024-02-19 17:11:18.135 UTC [99] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:18.135 UTC [99] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:18.162 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:18.162 UTC [88] DETAIL:  Failing row contains (1, 1, -102898, 100000).
postgres-1  | 2024-02-19 17:11:18.162 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:18.162 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:18.215 UTC [90] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:18.215 UTC [90] DETAIL:  Failing row contains (1, 1, -103872, 100000).
postgres-1  | 2024-02-19 17:11:18.215 UTC [90] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:18.215 UTC [90] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:18.378 UTC [68] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:18.378 UTC [68] DETAIL:  Failing row contains (1, 1, -100106, 100000).
postgres-1  | 2024-02-19 17:11:18.378 UTC [68] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:18.378 UTC [68] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:18.860 UTC [100] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:18.860 UTC [100] DETAIL:  Failing row contains (2, 2, -80815, 80000).
postgres-1  | 2024-02-19 17:11:18.860 UTC [100] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:18.860 UTC [100] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:18.913 UTC [99] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:18.913 UTC [99] DETAIL:  Failing row contains (2, 2, -80064, 80000).
postgres-1  | 2024-02-19 17:11:18.913 UTC [99] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:18.913 UTC [99] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:18.939 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:18.939 UTC [88] DETAIL:  Failing row contains (2, 2, -85078, 80000).
postgres-1  | 2024-02-19 17:11:18.939 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:18.939 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:19.878 UTC [99] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:19.878 UTC [99] DETAIL:  Failing row contains (2, 2, -80597, 80000).
postgres-1  | 2024-02-19 17:11:19.878 UTC [99] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:19.878 UTC [99] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:20.028 UTC [90] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:20.028 UTC [90] DETAIL:  Failing row contains (2, 2, -80639, 80000).
postgres-1  | 2024-02-19 17:11:20.028 UTC [90] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:20.028 UTC [90] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:20.053 UTC [68] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:20.053 UTC [68] DETAIL:  Failing row contains (2, 2, -84264, 80000).
postgres-1  | 2024-02-19 17:11:20.053 UTC [68] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:20.053 UTC [68] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:20.128 UTC [89] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:20.128 UTC [89] DETAIL:  Failing row contains (2, 2, -82660, 80000).
postgres-1  | 2024-02-19 17:11:20.128 UTC [89] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:20.128 UTC [89] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:20.203 UTC [77] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:20.203 UTC [77] DETAIL:  Failing row contains (2, 2, -81662, 80000).
postgres-1  | 2024-02-19 17:11:20.203 UTC [77] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:20.203 UTC [77] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:20.353 UTC [99] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:20.353 UTC [99] DETAIL:  Failing row contains (2, 2, -81446, 80000).
postgres-1  | 2024-02-19 17:11:20.353 UTC [99] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:20.353 UTC [99] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:20.453 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:20.453 UTC [88] DETAIL:  Failing row contains (2, 2, -82113, 80000).
postgres-1  | 2024-02-19 17:11:20.453 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:20.453 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:20.616 UTC [68] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:20.616 UTC [68] DETAIL:  Failing row contains (1, 1, -101304, 100000).
postgres-1  | 2024-02-19 17:11:20.616 UTC [68] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:20.616 UTC [68] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:21.226 UTC [100] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:21.226 UTC [100] DETAIL:  Failing row contains (2, 2, -80351, 80000).
postgres-1  | 2024-02-19 17:11:21.226 UTC [100] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:21.226 UTC [100] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:21.298 UTC [68] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:21.298 UTC [68] DETAIL:  Failing row contains (1, 1, -101861, 100000).
postgres-1  | 2024-02-19 17:11:21.298 UTC [68] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:21.298 UTC [68] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:21.322 UTC [90] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:21.322 UTC [90] DETAIL:  Failing row contains (1, 1, -103196, 100000).
postgres-1  | 2024-02-19 17:11:21.322 UTC [90] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:21.322 UTC [90] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:21.560 UTC [68] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:21.560 UTC [68] DETAIL:  Failing row contains (1, 1, -102364, 100000).
postgres-1  | 2024-02-19 17:11:21.560 UTC [68] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:21.560 UTC [68] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:21.583 UTC [99] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:21.583 UTC [99] DETAIL:  Failing row contains (1, 1, -101699, 100000).
postgres-1  | 2024-02-19 17:11:21.583 UTC [99] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:21.583 UTC [99] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:21.747 UTC [68] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:21.747 UTC [68] DETAIL:  Failing row contains (1, 1, -100593, 100000).
postgres-1  | 2024-02-19 17:11:21.747 UTC [68] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:21.747 UTC [68] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:21.998 UTC [68] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:21.998 UTC [68] DETAIL:  Failing row contains (1, 1, -101593, 100000).
postgres-1  | 2024-02-19 17:11:21.998 UTC [68] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:21.998 UTC [68] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:22.089 UTC [77] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:22.089 UTC [77] DETAIL:  Failing row contains (1, 1, -102693, 100000).
postgres-1  | 2024-02-19 17:11:22.089 UTC [77] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:22.089 UTC [77] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:22.156 UTC [89] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:22.156 UTC [89] DETAIL:  Failing row contains (2, 2, -82364, 80000).
postgres-1  | 2024-02-19 17:11:22.156 UTC [89] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:22.156 UTC [89] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:22.292 UTC [100] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:22.292 UTC [100] DETAIL:  Failing row contains (2, 2, -84024, 80000).
postgres-1  | 2024-02-19 17:11:22.292 UTC [100] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:22.292 UTC [100] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:22.316 UTC [87] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:22.316 UTC [87] DETAIL:  Failing row contains (2, 2, -80619, 80000).
postgres-1  | 2024-02-19 17:11:22.316 UTC [87] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:22.316 UTC [87] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:22.474 UTC [89] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:22.474 UTC [89] DETAIL:  Failing row contains (2, 2, -86733, 80000).
postgres-1  | 2024-02-19 17:11:22.474 UTC [89] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:22.474 UTC [89] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:22.498 UTC [90] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:22.498 UTC [90] DETAIL:  Failing row contains (2, 2, -84969, 80000).
postgres-1  | 2024-02-19 17:11:22.498 UTC [90] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:22.498 UTC [90] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:22.521 UTC [100] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:22.521 UTC [100] DETAIL:  Failing row contains (2, 2, -86262, 80000).
postgres-1  | 2024-02-19 17:11:22.521 UTC [100] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:22.521 UTC [100] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:22.589 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:22.589 UTC [88] DETAIL:  Failing row contains (1, 1, -100485, 100000).
postgres-1  | 2024-02-19 17:11:22.589 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:22.589 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:22.713 UTC [142] FATAL:  role "root" does not exist
postgres-1  | 2024-02-19 17:11:22.962 UTC [100] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:22.962 UTC [100] DETAIL:  Failing row contains (2, 2, -87394, 80000).
postgres-1  | 2024-02-19 17:11:22.962 UTC [100] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:22.962 UTC [100] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:23.093 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:23.093 UTC [88] DETAIL:  Failing row contains (2, 2, -86305, 80000).
postgres-1  | 2024-02-19 17:11:23.093 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:23.093 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:23.202 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:23.202 UTC [88] DETAIL:  Failing row contains (2, 2, -83151, 80000).
postgres-1  | 2024-02-19 17:11:23.202 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:23.202 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:23.397 UTC [87] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:23.397 UTC [87] DETAIL:  Failing row contains (2, 2, -81966, 80000).
postgres-1  | 2024-02-19 17:11:23.397 UTC [87] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:23.397 UTC [87] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:23.441 UTC [100] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:23.441 UTC [100] DETAIL:  Failing row contains (1, 1, -107899, 100000).
postgres-1  | 2024-02-19 17:11:23.441 UTC [100] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:23.441 UTC [100] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:23.462 UTC [68] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:23.462 UTC [68] DETAIL:  Failing row contains (1, 1, -104284, 100000).
postgres-1  | 2024-02-19 17:11:23.462 UTC [68] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:23.462 UTC [68] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:23.659 UTC [90] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:23.659 UTC [90] DETAIL:  Failing row contains (1, 1, -101352, 100000).
postgres-1  | 2024-02-19 17:11:23.659 UTC [90] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:23.659 UTC [90] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:23.744 UTC [90] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:23.744 UTC [90] DETAIL:  Failing row contains (2, 2, -85054, 80000).
postgres-1  | 2024-02-19 17:11:23.744 UTC [90] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-19 17:11:23.744 UTC [90] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-19 17:11:24.042 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-19 17:11:24.042 UTC [88] DETAIL:  Failing row contains (1, 1, -106169, 100000).
